ffs/api/api.go

```diff
@@ -128,7 +172,7 @@ func (i *API) SetDefaultCidConfig(c ffs.DefaultCidConfig) error {
 		return fmt.Errorf("default cid config is invalid: %s", err)
 	}
 	i.cfg.DefaultCidConfig = c
-	return nil
+	return i.is.PutConfig(i.cfg)
```

Comment:
I think this was a bug in the wild... the newly set `DefaultCidConfig` wasn't being persisted, so just added that.

Reply:
Yep.